pkgs/applications/misc/etesync-dav/default.nix

```diff
@@ -0,0 +1,76 @@
+{ lib, python3, radicale2 }:
+with python3.pkgs;
+
+let
+  etesync = buildPythonPackage rec {
+    pname = "etesync";
+    version = "0.9.2";
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "03016nwn67b0nk8gl6ycnz5c7b8jnxgyrypx9zv5sfara706xmd8";
+    };
+
+    propagatedBuildInputs = [
+      appdirs
+      asn1crypto
+      cffi
+      cryptography
+      furl
+      idna
+      orderedmultidict
+      packaging
+      peewee
+      py
+      pyasn1
+      pycparser
+      pyparsing
+      pyscrypt
+      python-dateutil
+      pytz
+      requests
+      six
+      vobject
+    ];
+
+    checkInputs = [
+      pytest
+    ];
+
+    checkPhase = ''
+      pytest tests/test_collections.py
+      pytest tests/test_crypto.py
+    '';
+  };
```

Comment:
personally, i would refactor this package into a `etesync.nix`, and just import it.

Reply:
Hi Jon, thanks for the quick review! I added all the other changes. I'm not sure how to do this one, though. Do you mean package it as a separate package, i.e. `etesync/default.nix`? Or actually have an `etesync.nix` in this one? If so, I'm not quite sure how to do that and couldn't find an example in another package to copy from.

Reply:
I just pulled it out into its own package :)